vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -620,6 +620,20 @@ def apply(self,
             raise ValueError("A scheme must be defined for each layer")
         return scheme.apply_weights(layer, x, bias=bias)
 
+    def dequant_fp8_weight(self,
+                           layer: "CompressedTensorsW8A8Fp8") -> torch.Tensor:
+        if layer.scheme.strategy == QuantizationStrategy.CHANNEL.value:
```

Comment:
"CompressedTensorsW8A8Fp8" has no attribute "scheme"  [attr-defined]"
why is this scheme from?

Reply:
fixed